x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_form.tsx

```diff
@@ -33,36 +33,130 @@ export const EditTransformFlyoutForm: FC<EditTransformFlyoutFormProps> = ({
           defaultMessage: 'Frequency',
         })}
         onChange={(value) => dispatch({ field: 'frequency', value })}
-        placeholder="1m"
+        placeholder={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormFrequencyPlaceholderText',
+          {
+            defaultMessage: 'Default: {defaultValue}',
+            values: { defaultValue: formFields.frequency.defaultValue },
+          }
+        )}
         value={formFields.frequency.value}
       />
+
+      <EuiSpacer size="l" />
+
+      <EuiAccordion
+        data-test-subj="transformEditAccordionDestination"
+        id="transformEditAccordionDestination"
+        buttonContent={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormDestinationButtonContent',
+          {
+            defaultMessage: 'Destination configuration',
+          }
+        )}
+        paddingSize="s"
+      >
+        <EditTransformFlyoutFormTextInput
+          dataTestSubj="transformEditFlyoutDestinationIndexInput"
+          errorMessages={formFields.destinationIndex.errorMessages}
+          label={i18n.translate(
+            'xpack.transform.transformList.editFlyoutFormDestinationIndexLabel',
+            {
+              defaultMessage: 'Destination index',
+            }
+          )}
+          onChange={(value) => dispatch({ field: 'destinationIndex', value })}
+          value={formFields.destinationIndex.value}
+        />
+
+        <EditTransformFlyoutFormTextInput
+          dataTestSubj="transformEditFlyoutDestinationPipelineInput"
+          errorMessages={formFields.destinationPipeline.errorMessages}
+          label={i18n.translate(
+            'xpack.transform.transformList.editFlyoutFormDestinationPipelineLabel',
+            {
+              defaultMessage: 'Pipeline',
+            }
+          )}
+          onChange={(value) => dispatch({ field: 'destinationPipeline', value })}
+          value={formFields.destinationPipeline.value}
+        />
+      </EuiAccordion>
+
+      <EuiSpacer size="l" />
+
+      <EuiAccordion
+        data-test-subj="transformEditAccordionAdvancedSettings"
+        id="transformEditAccordionAdvancedSettings"
+        buttonContent={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormAdvancedSettingsButtonContent',
+          {
+            defaultMessage: 'Advanced settings',
+          }
+        )}
+        paddingSize="s"
+      >
+        <div data-test-subj="transformEditAccordionAdvancedSettingsContent">
+          <EditTransformFlyoutFormTextInput
+            dataTestSubj="transformEditFlyoutDocsPerSecondInput"
+            errorMessages={formFields.docsPerSecond.errorMessages}
+            helpText={i18n.translate(
+              'xpack.transform.transformList.editFlyoutFormDocsPerSecondHelptext',
+              {
+                defaultMessage:
+                  'To enable throttling, set a limit of documents per second of input documents.',
```

Comment:
This could do with a rephrase to avoid repetition of  `documents`. `To enable throttling, set a limit of documents to input per second.` ?

Reply:
Fixed in 38d0cdb4e94.

Reply:
I think we need to use sentence case here and include the `.` at the end.

Reply:
Fixed in 719d6ad.